bcipy/signal/model/keyinput.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
+
+
+def compute_keyinput_probs(
+    inquiry: List[str],
+    symbol_set: List[str],
+    user_error_prob: float,
+    proceed: bool,
+):
```

Comment:
Add a return type
example a funtion that accepts a list of strings and returns a bool:

`def test(arg1: List[str]) -> bool:`